applications/nrf5340_audio/src/bluetooth/Kconfig.defaults

```diff
@@ -143,8 +142,13 @@ config BT_AUDIO_UNICAST_CLIENT
 	bool
 	default y
 
+config BT_ISO_TX_BUF_COUNT
+	int
+	default 2
+
 config BT_MAX_CONN
 	int
```

Comment:
I know this is existing code, but when update the default then please remove the `int` type.
```suggestion
```